tutorials/scripting/c_sharp/diagnostics/GD0002.rst

```diff
@@ -0,0 +1,74 @@
+GD0002: Missing partial modifier on declaration of type which contains one or more subclasses of GodotObject
```

Comment:
```suggestion
GD0002: Missing partial modifier on declaration of type which contains one or more subclass of GodotObject
```

(I think) But this is probably because the actual diagnostic.

Reply:
It just looked wrong to me, with some more thought, both versions are probably fine, but if we want to change it I would also go to a different wording like `contains nested classes that are derived from GodotObject`

Reply:
I like that wording. This change needs to be done in the main repository as well, is there any other diagnostic that you think could be worded better? It'd be nice to make all the changes in a single PR while we're at it.

Reply:
- I've opened https://github.com/godotengine/godot/pull/80489.